modules/dxp/apps/portal-workflow/portal-workflow-metrics-test/src/testFunctional/tests/WorkflowMetricsAllItemsList.testcase

```diff
@@ -1485,6 +1485,7 @@ definition {
 	@priority = 3
 	test IconOnModalTitleIsGreyWhenItemIsCompletedOnTime {
 		property test.liferay.virtual.instance = "false";
+		property portal.acceptance = "quarantine";
```

Comment:
These tests are consistently failing on CI after the test has been fixed due to timer/status.